### ISSUE TITLE
Toggling decimal/whole number ticks on axis

### ISSUE THREAD

**Issue by driaug** [tremorlabs/tremor-npm#253]
In the current setup, graphs automatically decide to show decimal ticks on charts when there is enough room. 
Certain scenarios will never have decimal values so it should be possible to either toggle decimal ticks or have it automatically decide based on the provided values.

![image](https://user-images.githubusercontent.com/31651090/208657154-b1cd9226-0343-44c2-8d2f-b8cc2d42c158.png)

**Comment by mitrotasios** [comment]
hey @driaug, thanks for the suggestion and sorry for the delayed response! I think that makes sense and we will include it in the following releases!

**Comment by mitrotasios** [comment]
hey @driaug, after testing out a couple of edge cases, we found out that this behaviour of rendering decimal numbers on the y-Axis is only achieved if the maximum value of your data is equal or less than 3. Because this is a very rare case (i.e. the need for having non-decimal discrete values up to exactly 3), we decided to not implement this property **for now** unfortunately, as we are afraid to pollute the namespace with too many minor flags. Sorry about that :( We will try to find a more general solution in the future though!
